framework/Simulation.py

```diff
@@ -284,34 +284,34 @@ def __init__(self,frameworkDir,verbosity='all',interactive=Interaction.No):
     #the keywords are the name of the module that contains the specialization of that specific entity
-    self.addWhatDict  = {}
-    self.addWhatDict['Steps'            ] = Steps
-    self.addWhatDict['DataObjects'      ] = DataObjects
-    self.addWhatDict['Samplers'         ] = Samplers
-    self.addWhatDict['Optimizers'       ] = Optimizers
-    self.addWhatDict['Models'           ] = Models
-    self.addWhatDict['Distributions'    ] = Distributions
-    self.addWhatDict['Databases'        ] = Databases
-    self.addWhatDict['Functions'        ] = Functions
-    self.addWhatDict['Files'            ] = Files
-    self.addWhatDict['Metrics'          ] = Metrics
-    self.addWhatDict['OutStreams'       ] = OutStreams
+    self.entityModules  = {}
+    self.entityModules['Steps'            ] = Steps
+    self.entityModules['DataObjects'      ] = DataObjects
+    self.entityModules['Samplers'         ] = Samplers
+    self.entityModules['Optimizers'       ] = Optimizers
+    self.entityModules['Models'           ] = Models
+    self.entityModules['Distributions'    ] = Distributions
+    self.entityModules['Databases'        ] = Databases
+    self.entityModules['Functions'        ] = Functions
+    self.entityModules['Files'            ] = Files
+    self.entityModules['Metrics'          ] = Metrics
+    self.entityModules['OutStreams'       ] = OutStreams
 
 
     #Mapping between an entity type and the dictionary containing the instances for the simulation
-    self.whichDict = {}
-    self.whichDict['Steps'           ] = self.stepsDict
-    self.whichDict['DataObjects'     ] = self.dataDict
-    self.whichDict['Samplers'        ] = self.samplersDict
-    self.whichDict['Optimizers'      ] = self.samplersDict
-    self.whichDict['Models'          ] = self.modelsDict
-    self.whichDict['RunInfo'         ] = self.runInfoDict
-    self.whichDict['Files'           ] = self.filesDict
-    self.whichDict['Distributions'   ] = self.distributionsDict
-    self.whichDict['Databases'       ] = self.dataBasesDict
-    self.whichDict['Functions'       ] = self.functionsDict
-    self.whichDict['Metrics'         ] = self.metricsDict
-    self.whichDict['OutStreams'      ] = self.outStreamsDict
+    self.entities = {}
+    self.entities['Steps'           ] = self.stepsDict
+    self.entities['DataObjects'     ] = self.dataDict
+    self.entities['Samplers'        ] = self.samplersDict
+    self.entities['Optimizers'      ] = self.samplersDict
+    self.entities['Models'          ] = self.modelsDict
+    self.entities['RunInfo'         ] = self.runInfoDict
+    self.entities['Files'           ] = self.filesDict
+    self.entities['Distributions'   ] = self.distributionsDict
+    self.entities['Databases'       ] = self.dataBasesDict
+    self.entities['Functions'       ] = self.functionsDict
+    self.entities['Metrics'         ] = self.metricsDict
+    self.entities['OutStreams'      ] = self.outStreamsDict
```

Comment:
This change is not functional for this PR. can you explain why you did it?

Reply:
I think you should just revert this... there are diffs that are not functional for this PR

Reply:
This is described in the PR description above:
```
- Renames obtuse variables in Simulation for easier comprehension and maintenance
```

Reply:
This change came because I was digging through Simulation trying to understand why the Databases were not initializing correctly, and the "addWhatDict" and "whichDict" terminology was impossible to follow without constantly looking at handwritten notes I made in my notebook. I realized that there's no reason to maintain this burden, and it was a very easy change to make in order to make sense of it for the next developers.

Reply:
> I think you should just revert this... there are diffs that are not functional for this PR

I made this change because of the inordinate amount of time it took to dig through loops of "addWhatDict" and "whichDict" trying to make the Databases work.

Is there a reason to retain the obtuse naming strategy?